hack/bazel-fmt.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2017 Red Hat, Inc.
+#
+
+set -e
+
+source hack/common.sh
+
+shfmt -i 4 -w ${KUBEVIRT_DIR}/cluster/ ${KUBEVIRT_DIR}/hack/ ${KUBEVIRT_DIR}/images/
+bazel run \
+    --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 \
+    --workspace_status_command=./hack/print-workspace-status.sh \
+    //:gazelle -- pkg/ tools/ tests/ cmd/
```

Comment:
maybe using `-exclude vendor` is more future prove?

Reply:
Hm, `-exclude` excludes the directory from being considered as import source when processing other files. I could not find an option, to not update files in a specific directory.